angular08thOct/src/app/angularmain/angularmain.component.html

```diff
@@ -0,0 +1,82 @@
+
+<html>
+<head>
+<title>table in html</title>
+   <style>
+        table
+           {
```

Comment:
move styles to scss

Reply:
ok, implemented the same similarly in the later projects